defs.bzl

```diff
@@ -43,6 +44,7 @@ pip_repository = repository_rule(
         "requirements": attr.label(allow_single_file=True, mandatory=True,),
         "wheel_env": attr.string_dict(),
         "python_interpreter": attr.string(default="python3"),
+        "timeout": attr.int(default = 600),
```

Comment:
nit: might be worth making this "unassigned" (negative value perhaps) by default so we do not redefine/override the default set by pip. If they ever change it, we will be pinned to 600

Reply:
This is a Bazel timeout I'm setting here. I can make that change though

Reply:
Oops, mb 😬, its up to you

Reply:
Gotta find what Bazel accepts as an "unassigned" value, otherwise I'll need two `.execute(..)` calls I think

Reply:
Can't find how the starlark object would convert a `None` value into a timeout. I'll add a comment and leave this.

Reply:
Well I tried `timeout=None` and got this weird error: 

```
		fail(<1 more arguments>)
error running 'git fetch origin refs/heads/*:refs/remotes/origin/* refs/tags/*:refs/tags/*' while working with @rules_cc:
fatal: unable to access 'https://github.com/bazelbuild/rules_cc/': getpeername() failed with errno 22: Invalid argument
ERROR: Analysis of target '//python/auth:unit_test' failed; build aborted: com.google.devtools.build.lib.packages.RepositoryFetchException: no such package '@rules_cc//cc': Traceback (most recent call last):
```

Not sure if it's a coincidence.